packages/context/src/binding-filter.ts

```diff
@@ -224,9 +224,13 @@ export function filterByKey(
   if (typeof keyPattern === 'string') {
     const regex = wildcardToRegExp(keyPattern);
     return binding => regex.test(binding.key);
-  } else if (keyPattern instanceof RegExp) {
+  }
+
+  if (keyPattern instanceof RegExp) {
     return binding => keyPattern.test(binding.key);
-  } else if (typeof keyPattern === 'function') {
+  }
+
+  if (typeof keyPattern === 'function') {
```

Comment:
Not sure if these changes are relevant.

Reply:
These are not. I may undo what can be considered a style change, if you want.

Reply:
Let's revert it for this PR.

Reply:
Just done it. Reverted.